frontend/graphql/generated.graphql

```diff
@@ -98,6 +98,46 @@ enum ENUM_MEMBER_ORGANISATION {
   SAMI
 }
 
```

Comment:
Looking through the generated type definitions and everything looks really good to me, thanks for adding these